README.md

```diff
@@ -28,6 +28,26 @@ The search lib is modular and currently implements the following layers:
 
 ![Search Lib Architecture](./docs/images/layer_cake.png)
 
+## Versioning & release policy
+
+Each Vind released is labeled in the repository with a tag fitting the following schema: _vind-va.b.c_ where 
+**vind-v** just points out that this belongs to a Vind version and **a.b.c** represents the release number, also used 
+for the artifact version. Prior to **vind-v1.3.0** there was not a clear policy on versioning.
+
+From __1.3.0__ the releases will strictly stick to the following diagram:
```

Comment:
I would remove the *the* and add *on*, so -> From __1.3.0__ on releases will strictly stick to the following diagram: